api/controllers/ResumeController.ts

```diff
@@ -0,0 +1,25 @@
+import { UserAuthentication } from "../middleware/UserAuthentication";
+import { JsonController, Params, Get, UseBefore } from "routing-controllers";
+import ResumeService from "../../services/ResumeService";
+import { UserModel } from "../../models/UserModel";
+import { UuidParam } from "../validators/GenericRequests";
+import { AuthenticatedUser } from "../decorators/AuthenticatedUser";
+import { GetResumesListResponse } from "../../types";
+
+@UseBefore(UserAuthentication)
+@JsonController('/resume')
+export class ResumeController {
+  
+  private resumeService: ResumeService;
+
+  constructor(resumeService: ResumeService) {
+    this.resumeService = resumeService;
+  }
+
+  @Get()
+  async getAllVisibleResumes(@Params() params: UuidParam,
+  @AuthenticatedUser() currentUser: UserModel): Promise<GetResumesListResponse> {
+    const resumes = await this.resumeService.getVisibleResumes();
+    return { error: null, resumes };
+  }
```

Comment:
I think we should have some kind of permissions check here. e.g., only admins should be able to get all visible resumes, not any member.